### ISSUE TITLE
Chrome DevTools

### ISSUE THREAD

**Issue by r58Playz** [catppuccin/catppuccin#2341]
### Link to repository

https://github.com/r58Playz/ctp-devtools.git

### Screenshots

<details>
<summary>Latte</summary>

![latte](https://github.com/catppuccin/catppuccin/assets/45221816/de997a9b-2158-4cff-a6ee-c79fe3b35ff1)

</details>
<details>
<summary>Frappe</summary>

![frappe](https://github.com/catppuccin/catppuccin/assets/45221816/d71765f3-606c-47ff-a624-12203438f2fb)

</details>
<details>
<summary>Macchiato</summary>

![macchiato](https://github.com/catppuccin/catppuccin/assets/45221816/e9c09748-327d-4e98-b070-f238889eee86)

</details>
<details>
<summary>Mocha</summary>

![mocha](https://github.com/catppuccin/catppuccin/assets/45221816/621e5eef-6208-4462-bdca-fdcc87d6c453)

</details>

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey :wave:, sorry for the wait on reviewing this!

A few comments from me:

1. Please update the outdated [`icons`](https://github.com/r58Playz/ctp-devtools/tree/main/template/images) directory with our [latest icon](https://github.com/catppuccin/catppuccin/blob/main/assets/logos/exports/1544x1544_circle.png).
2. As far as I can see, the only files that change from the template directory are [`devtools.css.hbr`](https://github.com/r58Playz/ctp-devtools/blob/main/template/devtools.css.hbr) and [`manifest.json.hbr`](https://github.com/r58Playz/ctp-devtools/blob/main/template/manifest.json.hbr). I believe you should be able to symlink the `icons` directory, `devtools.html`, and `devtools.js` into the `themes` directory. This will massively reduce the amount of duplicated files and I think users should still be able to download the theme fine.
3. Since this repository was created, we have since released [whiskers v2](https://github.com/catppuccin/toolbox/tree/main/whiskers) which should reduce the complexity in the `generate.sh` script and the template files, consider updating the repository to use it! 

@uncenter I believe you tested this out earlier, did you notice any issues that are worth raising?

**Comment by uncenter** [comment]
> @uncenter I believe you tested this out earlier, did you notice any issues that are worth raising?

It works great! Sometimes it doesn't get loaded though, but I think that is Chrome's fault and nothing we can do about it.

**Comment by r58Playz** [comment]
Updated according to the suggestions: commit [`1f9e119`](https://github.com/r58Playz/ctp-devtools/commit/1f9e119ab786e0433ae0a7ff99f6a9e6741735af)

**Comment by sgoudham** [comment]
Thanks for implementing my suggestions! @r58Playz, sorry for the wait on getting back to this.

Great work, happy to approve this to get merged into the organisation. I think it makes sense to transfer it under the name `catppuccin/chome-devtools`.

Please read, ["Transferring a repository owned by your personal account"](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-personal-account), if you are unsure about the process of transferring the repository to Catppuccin.

**We'll close this issue once the port has been added to our [ports.yml](https://github.com/catppuccin/catppuccin/blob/main/resources/ports.yml)**.
